contracts/Orchestrator.sol

```diff
@@ -47,7 +47,7 @@ contract Orchestrator is Ownable {
         for (uint256 i = 0; i < transactions.length; i++) {
             Transaction storage t = transactions[i];
             if (t.enabled) {
-                bool result = externalCall(t.destination, t.data);
+                (bool result, bytes memory reason) = t.destination.call(t.data);
                 if (!result) {
                     emit TransactionFailed(t.destination, i, t.data);
```

Comment:
With the new flow, if one of the transactions reverts rebase reverts. 

The `TransactionFailed` log message will not be emitted when transaction reverts, it can be removed.

Reply:
If I remove the log message, should I also remove the `TransactionFailed` event or do we listen to that event anywhere else? https://github.com/ampleforth/uFragments/blob/07e229b2c3dd23b1b55ed871207196179fdfb5e6/contracts/Orchestrator.sol#L19

Reply:
I think we can remove it. The contract doesn't really use it so there's not need for it.

Services do listen to this, however the next orchestrator version will be a fresh contract deployment so it shouldn't break anything :)

Reply:
awesome will remove it.